dev/local-environment/configurations/pc-contracts-cli/entrypoint.sh

```diff
@@ -107,20 +109,17 @@ echo "Generating addresses.json file..."
 
 export COMMITTEE_CANDIDATE_ADDRESS=$(jq -r '.addresses.CommitteeCandidateValidator' addresses.json)
 echo "Committee candidate address: $COMMITTEE_CANDIDATE_ADDRESS"
-echo COMMITTEE_CANDIDATE_ADDRESS=$COMMITTEE_CANDIDATE_ADDRESS > /shared/COMMITTEE_CANDIDATE_ADDRESS
 
 export D_PARAMETER_POLICY_ID=$(jq -r '.mintingPolicies.DParameterPolicy' addresses.json)
 echo "D parameter policy ID: $D_PARAMETER_POLICY_ID"
-echo D_PARAMETER_POLICY_ID=$D_PARAMETER_POLICY_ID > /shared/D_PARAMETER_POLICY_ID
 
 export PERMISSIONED_CANDIDATES_POLICY_ID=$(jq -r '.mintingPolicies.PermissionedCandidatesPolicy' addresses.json)
 echo "Permissioned candidates policy ID: $PERMISSIONED_CANDIDATES_POLICY_ID"
-echo PERMISSIONED_CANDIDATES_POLICY_ID=$PERMISSIONED_CANDIDATES_POLICY_ID > /shared/PERMISSIONED_CANDIDATES_POLICY_ID
 
-echo "Importing environment variables from shared files..."
-export NATIVE_TOKEN_POLICY_ID=$(cat /shared/NATIVE_TOKEN_POLICY_ID)
-export NATIVE_TOKEN_ASSET_NAME=$(cat /shared/NATIVE_TOKEN_ASSET_NAME)
-export ILLIQUID_SUPPLY_VALIDATOR_ADDRESS=$(cat /shared/ILLIQUID_SUPPLY_VALIDATOR_ADDRESS)
+echo "Setting placeholder values for NATIVE_TOKEN_POLICY_ID, NATIVE_TOKEN_ASSET_NAME, and ILLIQUID_SUPPLY_VALIDATOR_ADDRESS for chain-spec creation"
```

Comment:
Replaces writing them to files in cardano node and reading here, because it's unnecessary complexity